js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -229,6 +251,8 @@
         } else {
           this.createEditGraphModal();
           // hide tab entries
+          // no EnterpriseGraphs in single server mode
+          $('#tab-enterpriseGraph').parent().remove();
```

Comment:
I can still see this tab in single server mode. When trying to save the graph, it says "smart graph attribute not given".

Reply:
Same error message for cluster as well.

Reply:
Oddly enough, same error message when trying to create a smart graph on a cluster as well (despite providing SGA).

Reply:
Regarding:
> Same error message for cluster as well.

and:
> Oddly enough, same error message when trying to create a smart graph on a cluster as well (despite providing SGA).

This is indeed not working as you need the backend of branch: `feature/enterprise-graphs` to test it properly. 

> I can still see this tab in single server mode. When trying to save the graph, it says "smart graph attribute not given". 

Regarding single server creation, I'll double check.

Reply:
>I can still see this tab in single server mode. When trying to save the graph, it says "smart graph attribute not given".

^-- Just confirmed: Works if you test against the proper backend (with feature included). This PR is only the UI.

Reply:
Still the comment is misleading. Changed to:
```
// no EnterpriseGraphs in the community edition.
```